### PR TITLE
react-883-survey-form-answer-screen

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Create View Survey Form / Answer Survey Screen
[#883](https://github.com/nerubia/kh360-react/issues/883)

### CODE REVIEW

File: src/features/survey/[id]/survey.tsx
Comment (emmaacha) [inline src/features/survey/[id]/survey.tsx:1]: Please rename to `survey-form` and follow the usual setup for our screens: 
- add survey-form.tsx file in pages under survey-forms (right now the folder name is is `survey` but can you rename to `survey-forms`?)/[id]. 
- Then add `survey-form-header.tsx`, and `survey-form-list.tsx` <- where we can add the questions list
- Please also rename folder inside features from `survey` `to survey-forms`

File: src/routes/routes.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ export const routes = {
       myEvaluations: "/my-evaluations",
       myEvaluationResults: "/my-evaluations/:id",
       surveyForms: "/survey-forms",
+      survey: "survey_administration/:id/details",
```
Comment (emmaacha) [inline src/routes/routes.ts:73]: Maybe we can change this to `viewSurveyForm: "/survey-forms/:id"`

**Review by emmaacha — CHANGES_REQUESTED** [review]

**Review by emmaacha — APPROVED** [review]

File: src/features/survey-forms/[id]/survey-form.tsx [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+        image: getRandomImage(),
+      },
+    ],
+  },
+]
+
+export default function SurveyForm() {
+  return (
+    <div>
+      <PageTitle>Test Survey Sample</PageTitle>
```
Comment (gborgonia-ir) [inline src/features/survey-forms/[id]/survey-form.tsx:251]: this should be inside survey-form-header?

File: src/pages/survey-forms/[id]/survey-form-header.tsx
Comment (emmaacha) [inline src/pages/survey-forms/[id]/survey-form-header.tsx:1]: This should be in features

File: src/pages/survey-forms/[id]/survey-form-list.tsx
Comment (emmaacha) [inline src/pages/survey-forms/[id]/survey-form-list.tsx:1]: This should be in features

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/features/survey-forms/[id]/survey-form.tsx (hunk 2 above)
Comment (chadiegilnerubia) [reply src/features/survey-forms/[id]/survey-form.tsx:251]: Sure, I'll move it over there.

File: src/pages/survey-forms/[id]/survey-form-header.tsx
Comment (chadiegilnerubia) [reply src/pages/survey-forms/[id]/survey-form-header.tsx:1]: Okey i'll move it mb

File: src/pages/survey-forms/[id]/survey-form-list.tsx
Comment (chadiegilnerubia) [reply src/pages/survey-forms/[id]/survey-form-list.tsx:1]: Okey i'll move it mb

File: src/pages/survey-forms/survey-forms.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { SurveyFormsHeader } from "@features/survey/survey-forms-header"
-import { SurveyFormsList } from "@features/survey/survey-forms-list"
+import { SurveyFormsHeader } from "@features/survey-forms/survey-forms-header"
+import { SurveyFormsList } from "@features/survey-forms/survey-forms-list"
 import { useTitle } from "@hooks/useTitle"
 
 export default function EvaluationAdministrations() {
```
Comment (emmaacha) [inline src/pages/survey-forms/survey-forms.tsx:5]: Just noticed this now, please change to `SurveyForms()`

File: src/features/survey-forms/[id]/survey-form.tsx
Comment (emmaacha) [inline src/features/survey-forms/[id]/survey-form.tsx:1]: `survey-form.tsx` should be the parent page, and should be moved inside `pages/survey-forms/[id]`, with its components broken down into children (`survey-form-header.tsx`, `survey-form-questions.tsx`, etc.), while the ones on the features folder, specifically for `features/survey-forms/[id]`, should be the children components, so `survey-form-header.tsx`, `survey-form-questions.tsx`, etc. should be inside this folder

File: src/features/survey-forms/survey-forms-header.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { PageTitle } from "@components/shared/page-title"
 export const SurveyFormsHeader = () => {
   return (
     <div className='flex flex-col md:flex-row justify-between gap-4'>
-      <PageTitle>Survey Forms</PageTitle>
+      <PageTitle>Test Survey Sample</PageTitle>
```
Comment (emmaacha) [inline src/features/survey-forms/survey-forms-header.tsx:6]: This header is for the survey forms (the list) page, please create another file for the view survey form header inside features/survey-forms/[id]

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/pages/survey-forms/survey-forms.tsx (hunk 3 above)
Comment (chadiegilnerubia) [reply src/pages/survey-forms/survey-forms.tsx:5]: oww yeahh mb xD

File: src/features/survey-forms/survey-forms-header.tsx (hunk 4 above)
Comment (chadiegilnerubia) [reply src/features/survey-forms/survey-forms-header.tsx:6]: done

**Comment by gborgonia-ir** [comment]
@chadiegilnerubia can pls also add these changes based from earlier discussion:
- show default image if no image from response
- checkbox should be beside the item name

**Comment by chadiegilnerubia** [comment]
> @chadiegilnerubia can pls also add these changes based from earlier discussion:
> 
> * show default image if no image from response
> * checkbox should be beside the item name

done :3

**Review by gborgonia-ir — APPROVED** [review]

**Review by emmaacha — APPROVED** [review]

**Pull request merged**